Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:24-slim
+
+WORKDIR /opt/app
+
+COPY server/target/unitycatalog-server-assembly.jar unitycatalog-server.jar
+COPY examples/cli/target/unitycatalog-cli-assembly.jar unitycatalog-cli.jar
```

Comment:
whats the purpose of the CLI jar in docker? we are not running the CLI in docker